src/module.c

```diff
@@ -203,6 +203,10 @@ static int parseRangeArguments(RedisModuleCtx *ctx, Series *series, int start_in
 static int parseCountArgument(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, long long *count) {
     int offset = RMUtil_ArgIndex("COUNT", argv, argc);
     if (offset > 0) {
+        if (offset + 1 == argc) {
+            RedisModule_ReplyWithError(ctx, "TSDB: COUNT argument is missing");
```

Comment:
@ashtul should we be this specific, or use the wrong arity one?

Reply:
@filipecosta90 we use explicit language quite often to help users understand their error.

Reply:
@filipecosta90 I agree that wrong arity is more "accurate" error, but its not like RESP has specific error code. I do like the more explicitness since our commands are getting more and more complex and harder to debug when theres error.

Reply:
agree. lets unblock this review (since it fixes an crash) and merge :+1: